src/main/java/org/mule/tools/maven/mojo/ProcessSourcesMojo.java

```diff
@@ -46,6 +47,9 @@
 
   @Override
   public void execute() throws MojoExecutionException, MojoFailureException {
+    long start = System.currentTimeMillis();
+    getLog().debug("Processing sources...");
```

Comment:
In this case is different because the flag is just to lightweightPackage package, the log is still useful in order to measure the whole mojo.